ui/src/sign-up-module/sign-up-page/sign-up-page.component.sass

```diff
@@ -0,0 +1,60 @@
+.main
+    display: flex
+    margin-bottom: 10px
+
+    @media (max-width: 767px)
+        .main
+            flex-direction: column
+
+    .loginText
+        width: 50%
+        margin-top: 40px
+        padding-right: 50px
+
+    @media (max-width: 767px)
+        .loginText
+            width: 100%
+            justify-content: center
+
+    .formFlex
+        display: flex
+        flex-direction: column
+
+    .inputBox
+        border: none
+        border-bottom: 1px solid grey
+
+    input
+        width: 250px
+        display: block
+
+    input:focus
+
+        outline: none
+        border: none
+        border-bottom: 2px solid #0b5c60
+
+    input:focus::placeholder
```

Comment:
Signup and sign-in share a lot of visual property. Please use common CSS to apply similar styles

Reply:
Okay. Resolved the issue as suggested.